auth-service/src/main/kotlin/uk/gov/communities/delta/auth/plugins/Templating.kt

```diff
@@ -2,24 +2,32 @@ package uk.gov.communities.delta.auth.plugins
 
 import io.ktor.server.application.*
 import io.ktor.server.thymeleaf.*
+import org.slf4j.LoggerFactory
+import org.thymeleaf.templateresolver.AbstractConfigurableTemplateResolver
 import org.thymeleaf.templateresolver.ClassLoaderTemplateResolver
 import org.thymeleaf.templateresolver.FileTemplateResolver
 
 fun Application.configureTemplating(devMode: Boolean) {
     install(Thymeleaf) {
-        setTemplateResolver((if (devMode) {
-            log.info("Development mode, using templates from src/ folder")
-            FileTemplateResolver().apply {
-                cacheManager = null
-                prefix = "auth-service/src/main/resources/templates/thymeleaf/"
-            }
-        } else {
-            ClassLoaderTemplateResolver().apply {
-                prefix = "templates/thymeleaf/"
-            }
-        }).apply {
-            suffix = ".html"
-            characterEncoding = "utf-8"
-        })
+        setTemplateResolver(makeTemplateResolver(devMode))
+    }
+}
+
+fun makeTemplateResolver(devMode: Boolean): AbstractConfigurableTemplateResolver {
+    val logger = LoggerFactory.getLogger("Application.Templating")
+
+    return (if (devMode) {
+        logger.info("Development mode, using templates from src/ folder")
+        FileTemplateResolver().apply {
+//            cacheManager = null // TODO - add this back in?
```

Comment:
Did it break something? (we don't want to cache them in dev mode so changes get picked up quickly)

Reply:
It just didn't recognise itself as an option 🤷 😂